src/main/java/command/Command.java

```diff
@@ -0,0 +1,60 @@
+package command;
+
+import java.io.IOException;
+import exception.CommandNotFoundException;
+import exception.InsufficentArgumentsException;
+import exception.TaskNotFoundException;
+import task.TaskManager;
+
+public class Command implements CommandInterface {
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_MARK_TASK = "mark";
+    private static final String COMMAND_UNMARK_TASK = "unmark";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_DELETE_TASK = "delete";
+
+    private static TaskManager taskManager = new TaskManager();
+
+    private final String command;
+    private final String[] args;
```

Comment:
Maybe consider naming this variable with something that better describes its purpose.